### DISCUSSION TITLE
carla strategy

### DISCUSSION THREAD

**Discussion by phkarle** [carla-simulator/carla#6025] [General]
This is Phillip from the Technical University of Munich, winner of the Indy Autonomous Challenge 2021. Our new simulation framework for public traffic is based on the carla simulator (AV Software: Autoware). So, I would like to get in touch with you to discuss some strategical aspects. Would be great if someone would be available for a meeting with me and my team.

**Comment by fghydx** [comment]
这是来自QQ邮箱的假期自动回复邮件。 您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
